test/input-validation.test.js

```diff
@@ -69,10 +69,13 @@ test('not evaluate json-schema $schema keyword', t => {
 })
 
 test('validation context in validation result', t => {
-  t.plan(2)
+  t.plan(3)
   const fastify = Fastify()
-  // return validation context, so we can test it in response
-  fastify.setErrorHandler((err, request, reply) => reply.send(err.validationContext))
+  // custom error handler to expose validation context in response, so we can test it later
+  fastify.setErrorHandler((err, request, reply) => {
+    t.equal(err instanceof Error, true)
```

Comment:
this assertion should verify that the `err.validationContext` property matches what we would expect.

Reply:
@mcollina oh, I see, sorry for taking so long to figure out ;-)... I was testing that later in the response itself, but I will test right there...